### ISSUE TITLE
[Keyboard navigation - Make Code - Advance] Buttons controls are not accessible using keyboard under 'Advance' expandible control.

### ISSUE THREAD

**Issue by sonimishra** [microsoft/pxt-microbit#2376]
**User Experience:**
User rely on keyboard will find difficulty to interact with the control if that control is not accessible through keyboard and won't be able to perform functionality over such control.

**Repro Steps:**
1. Launch application URL : https://makecode.microbit.org/beta#
2. Home page will open.
3. Navigate "New Project" option and select it.
4. Editor screen will open.
5. Navigate to "Arrays, Text, Games, Image" controls appearing under advance section and select it.
6. Navigate through all the controls appearing after selection of elements.
7. Verify the issue.

**Actual Result:**
Arrays, Text, Games, Image, Pin, Serial, Control under Advanced expandable controls are not accessible using keyboard.

**Expected Result:**
Arrays, Text, Games, Image, Pin, Serial, Control under Advanced expandable controls should be accessible using keyboard.

**Note:**
Same issue is also exists with Chrome.

**MAS Reference:**
MAS 2.1.1 - Keyboard

Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.

[MAS2.1.1_Controls are not accessible using keyboard.zip](https://github.com/microsoft/pxt-microbit/files/3481308/MAS2.1.1_Controls.are.not.accessible.using.keyboard.zip)

**Comment by jwunderl** [comment]
![2019-08-08 10 43 49](https://user-images.githubusercontent.com/5615930/62725220-915f0780-b9c9-11e9-87d9-91933e26ecc7.gif)

This is what I see when I try to navigate through the options; the advanced category does expand to more option for me; is there a different expected behavior here?

**Comment by jwunderl** [comment]
Duplicate of https://github.com/microsoft/pxt-microbit/issues/2378

**Comment by sonimishra** [comment]
Issue has been verified and seems product team has as exception for block part.
Hence closing the bug.
